src/transformers/convert_slow_tokenizer.py

```diff
@@ -46,7 +46,7 @@ def import_protobuf(error_message=""):
 def _get_prepend_scheme(add_prefix_space: bool, original_tokenizer) -> str:
     if add_prefix_space:
         prepend_scheme = "always"
-        if hasattr(original_tokenizer, "legacy") and not original_tokenizer.legacy:
+        if not getattr(original_tokenizer, "legacy", None):
```

Comment:
nit - if we're using `if not x` we should default to a boolean rather than a null

```suggestion
        if not getattr(original_tokenizer, "legacy", False):
```

Reply:
previously, if no `legacy` attribute, we won't enter the if's body.
now, if no `legacy` attribute, 

- `getattr(original_tokenizer, "legacy", None/False)` gives `None/False`
- `not getattr(original_tokenizer, "legacy", False)` gives `True`

so it will enter the if's body.

I am not sure this is expected ..?

Reply:
oups yeah then legacy should be set to false

Reply:
maybe it is expected though: if no `legacy` attribute, it means `it is not legacy` and now we always enter this body. I leave you to judge it as you are the expert!